cypress/e2e/checkout_spec.cy.js

```diff
@@ -1,63 +1,14 @@
-export const hasOperationName = (req, operationName) => {
-  const { body } = req
-  return (
-    body.hasOwnProperty('operationName') && body.operationName === operationName
-  )
-}
-
-// Alias query if operationName matches
-export const aliasQuery = (req, operationName) => {
-  if (hasOperationName(req, operationName)) {
-    req.alias = `gql${operationName}Query`
-  }
-}
-
-// Alias mutation if operationName matches
-export const aliasMutation = (req, operationName) => {
-  if (hasOperationName(req, operationName)) {
-    req.alias = `gql${operationName}Mutation`
-  }
-}
-
-const fixtureMap = {
-  GetAllItems: 'items.json',
-  CreateOrderForm: 'success.json'
-};
-
 describe('checkout', () => {
   beforeEach(() => {
-    cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/graphql*', (req) => {
-      // Queries
-      if (hasOperationName(req, 'GetAllItems')) {
-        aliasQuery(req, 'GetAllItems')
-        const fixtureName = fixtureMap['GetAllItems'];
-        req.reply((res) => {
-          res.send({
-            fixture: fixtureName, // Use the fixture for the response
-            statusCode: 201
-          });
-        });
-      }
-
-      // Mutations
-      if (hasOperationName(req, 'CreateOrderForm')) {
-        aliasMutation(req, 'CreateOrderForm')
-        const fixtureName = fixtureMap['CreateOrderForm'];
-        req.reply((res) => {
-          res.send({
-            fixture: fixtureName, // Use the fixture for the response
-            statusCode: 201
-          });
-        });
-      };
-    });
-
+    cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/graphql', {
+      statusCode: 200,
+      fixture: 'items.json'
+    }).as('getItems')
     cy.visit('http://localhost:3000/checkout')
+    cy.wait('@getItems')
   })
-
   it('All elements should be on the page and contain the correct values', () => {
-    cy.wait('@gqlGetAllItemsQuery')
-      .get('h2').first().contains('EverUse')
+    cy.get('h2').first().contains('EverUse')
```

Comment:
Non blocker but I use `eq(0)` instead of `first()`. This allows for dynamic references in child nodes. I could refer to any child by using `eq(elementNumber)`